x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -205,21 +205,22 @@ function useServicesDetailedStatisticsFetcher({
 }
 
 export function ServiceInventory() {
-  const { mainStatisticsFetch } = useServicesMainStatisticsFetcher();
+  const [debouncedSearchQuery, setDebouncedSearchQuery] = useStateDebounced(
+    '',
+    200
```

Comment:
We're using `300` for the useStateDebounced on the TransactionsTable. Maybe change here or there for consistency?

Reply:
Makes sense 👍